xcell/mappers/mapper_Planck_base.py

```diff
@@ -29,10 +29,10 @@ def _get_Planck_defaults(self, config):
     def get_signal_map(self):
         if self.signal_map is None:
             signal_map = hp.read_map(self.file_map)
+            # signal_map[signal_map == hp.UNSEEN] = 0.0
+            signal_map[np.isnan(signal_map)] = 0.0
```

Comment:
Hold on, what's going on here? Why did you comment out the first line?

Reply:
Uncommented